.github/workflows/security_tests_v2.yml

```diff
@@ -0,0 +1,189 @@
+name: V2 Security Tests
+
+on:
+  push:
+    branches: [ v2 ]
+  workflow_dispatch:
+    inputs:
+      docker-tag:
+        description: 'Docker tag for scan'
+        default: 'latest'
+        required: false
+
+jobs:
+  security-operator:
+    runs-on: ubuntu-latest
+    container: snyk/snyk:golang
+    steps:
+    - uses: actions/checkout@v2
+    - name: security-golang
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      run: |
+        make -C operator fmt
+        snyk test --file=operator/go.mod --fail-on=upgradable --severity-threshold=high
+
+  security-scheduler:
+    runs-on: ubuntu-latest
+    container: snyk/snyk:golang
+    steps:
+    - uses: actions/checkout@v2
+    - name: security-golang
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      run: |
+        make -C scheduler lint
```

Comment:
Do you know why need to run the linter here, instead of using Snyk's GH action?

Reply:
In all other tasks there was `make -C <directory> fmt` but `scheduler` did not have `fmt` but `lint` command. Not sure exactly why it is there. I first assumed it was for Go to pull modules but that does not make sense actually... I may try without it. 

But to why these are here: it's because these were present in V1 tasks.